src/utilities/modules/dispensary/constants.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+export const CASH_TRANSACTION_KEYS = {
```

Comment:
I don't like this only because I feel like a better way is to have something like `database/constants` file which would have things like status', all data types (so can do `UIDatabase.objects(DATA_TYPES.REQUISITION)` types etc. which I have been putting off. A bit whatever though

Reply:
I agree, just a temporary fix I think. Shouldn't be any issue with refactoring it in whatever way once we get time for a bit of tidy up.